salt/states/boto_lambda.py

```diff
@@ -489,7 +483,7 @@ def _function_config_present(
         )
         if not _r.get("updated"):
             ret["result"] = False
-            ret["comment"] = "Failed to update function: " "{0}.".format(
+            ret["comment"] = "Failed to update function: " "{}.".format(
```

Comment:
```suggestion
            ret["comment"] = "Failed to update function: {}.".format(
```
And all other occurrences like this one